src/main/java/frc/robot/ErrorCheck.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot;
```

Comment:
need to consider the location of this file

Reply:
CheckError is already in lib.lib2706 in main